lib/heroku_rails_deploy/deployer.rb

```diff
@@ -0,0 +1,103 @@
+require 'optparse'
+require 'yaml'
+
+module HerokuRailsDeploy
+  class Deployer
+    attr_reader :config_file, :args
+
+    class Options < Struct.new(:environment, :revision)
```

Comment:
@jturkel, Thoughts on "Don't extend an instance initialized by Struct.new. Extending it introduces a superfluous class level and may also introduce weird errors if the file is required multiple times." ? 

Ref: https://github.com/bbatsov/ruby-style-guide#no-extend-struct-new